src/main/java/DukeException.java

```diff
@@ -0,0 +1,107 @@
+package main.java;
+
+public class DukeException {
+    protected ExceptionType et;
+
+    public DukeException(ExceptionType et) {
+        this.et = et;
+    }
+
+    public enum ExceptionType {
+        no_meaning,
+        empty_illegal,
+        todo_empty,
+        deadline_empty_incomplete,
+        event_empty_incomplete,
+        read_file,
+        improper_dateTime
+    }
+
+    @Override
+    public String toString() {
+        String output = "";
+        String full_guide = "\n     Please follow the format:" +
+                "\n     'list'" + "\n     'done TASK_NUMBER' or 'delete TASK_NUMBER'" + "\n     'todo MY_TASK'" +
+                "\n     'deadline MY_TASK /by DATE_OR_TIME'" + "\n     'event MY_TASK /at DATE_OR_TIME'" +
+                "\n     'bye'";
+        switch(et) {
+            case no_meaning:
```

Comment:
You may adjust the indentation for switch method as there should not be any indentation between switch and case according to the guideline :)